test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -202,13 +324,10 @@ public function invalidVersionStringsProvider() : array
     {
         return [
             [''],
-            ['12.a'],
```

Comment:
Would replacing `a` with `z` (for example) allow us to retain these examples?

Reply:
probably you are right, will correct this

Reply:
you are so right, I added `z` ..and it broke so many test, fixing it now and probably tomorrow -_-

Reply:
Still applicable?

Reply:
yes, this invalid constraint was returned back )